### PR TITLE
feat(dist): empty template implementation of the cluster api

### PR DESCRIPTION
**Pull request by lenaschoenburg**
Adds a template for the cluster API. Actual methods are not yet implemented.
This also touches some related code to make sure that the coordinator starts the request handler.

Related to https://github.com/camunda/zeebe/issues/14462

### CODE REVIEW

File: dist/src/main/java/io/camunda/zeebe/shared/management/ClusterEndpoint.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  public WebEndpointResponse<?> addMember(
+      @Selector final ClusterComponent clusterComponent, @Selector final String id) {
+    return switch (clusterComponent) {
+      case ClusterComponent.BROKERS -> new WebEndpointResponse<>(
+          "Adding brokers is not supported", 400);
+    };
+  }
+
+  public enum ClusterComponent {
+    BROKERS,
```
Comment (deepthidevaki) [inline dist/src/main/java/io/camunda/zeebe/shared/management/ClusterEndpoint.java:47]: :+1: Didn't know this would work.

**Review by deepthidevaki — APPROVED** [review]
:rocket:

File: dist/src/main/java/io/camunda/zeebe/shared/management/ClusterEndpoint.java (hunk 1 above)
Comment (oleschoenburg) [reply dist/src/main/java/io/camunda/zeebe/shared/management/ClusterEndpoint.java:47]: Yeah I was pleasantly surprised too :D

**Comment by oleschoenburg** [comment]
bors merge

**Pull request merged**